src/Redux/store.js

```diff
@@ -1,12 +1,18 @@
-import { legacy_createStore as createStore, combineReducers, applyMiddleware } from 'redux';
+import {
+  legacy_createStore as createStore,
+  combineReducers,
+  applyMiddleware,
+} from 'redux';
 import thunk from 'redux-thunk';
 import logger from 'redux-logger';
 import roomsReducer from './Rooms/rooms';
 import reservationsReducer from './Reservations/Reservations';
+import handleUser from './Authenticate/authentication';
 
 const reducers = combineReducers({
   rooms: roomsReducer,
   reservations: reservationsReducer,
+  user: handleUser,
```

Comment:
**[OPTIONAL]** It would be great to be consistent with the other reducers' names. You can change this name to `usersReducer`

Reply:
- This Also was a refactor from previous code we did.  We might have to refactor things together, as they might break features.

Reply:
Yeah, for now, a change in the name here wont affect anything, considering the fact that is a default export.

Reply:
- okay, lets change this.